src/app/7-pipes/capital.pipe.ts

```diff
@@ -0,0 +1,34 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'capital'
+})
+export class CapitalPipe implements PipeTransform {
+
+  transform(val: string): string {
+    let allWords = val.includes(" ")? val.split(" "): [];
```

Comment:
Cambiar implementación a return val.split(" ").map(w => this.capitalizeText(w)).join(" ");